web/src/main/java/org/openmrs/web/controller/LoginController.java

```diff
@@ -127,6 +126,14 @@ public String handleRequest(WebRequest webRequest, ModelMap model) {
 			model.put("refererUrl", refererUrl);
 		}
 		
+		if (webRequest.getParameter(REDIRECT_URL) != null) {
+			String redirectUrlTemp = webRequest.getParameter(REDIRECT_URL).toString();
```

Comment:
Is this the only change necessary for this ticket? It would be easier to review this if you separated the cosmetic WebConstants. prefix changes from the actual fix for the reported problem. Does it make sense? :)